src/pybind/mgr/cephadm/module.py

```diff
@@ -1605,6 +1605,16 @@ def _find_inv_for_host(hostname, inventory_list):
 
         return self._create_osd(cmds)
 
+    def show_drivegroups(self):
+        # TODO: Currently exclusive to cephadm, maybe this can be moved to orchestrator.py once an agreement is made
+        return trivial_result(self.get_store("drivegroups"))
+
+    def set_drivegroups(self, content: str):
```

Comment:
What does `set` mean? `add` (just adding new dgs) or `apply` (also removes them)?

```suggestion
    def appy_drivegroups(self, content: List[DriveGroupSpec]):
```